### PR TITLE
Bugfix/compare profile page

### PR DESCRIPTION
**Pull request by rimashah25**
Compare Profile Page isn't working in TP. This PR fixes the same.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?

Login to TP, Click on Configure -> Profiles -> More -> Compare Profiles.
Select any two profiles and hit comapare.
Without this fix, the below page will not appear and the user is redirected to profile page (list of profiles in Table)
Wit this fix, the user will see the below page
image

## If this is a bugfix, which Traffic Control versions contained the bug?

7.1.0

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/profile/profiles.go [hunk 1]
```diff
@@ -439,9 +439,9 @@ func Read(w http.ResponseWriter, r *http.Request) {
 	profileInterfaces := []interface{}{}
 
 	for _, p := range profileList {
-		// Attach Parameters if the 'param' parameter is sent
-		if _, ok := inf.Params["param"]; ok {
-			p.Parameters, err = ReadParameters(inf.Tx, inf.User, p.ID, inf.User.Can("PARAMETER-SECURE:READ"))
+		// Attach Parameters if the 'id' parameter is sent
+		if _, ok := inf.Params["id"]; ok {
+			p.Parameters, err = ReadParameters(inf.Tx, inf.User, p.ID, inf.User.Can("PARAMETER:SECURE-READ"))
```
Comment (mitchell852) [inline traffic_ops/traffic_ops_golang/profile/profiles.go:444]: does this mean you need to have the PARAMETER:SECURE-READ permission to perform a profile compare?

Comment (rimashah25) [reply traffic_ops/traffic_ops_golang/profile/profiles.go:444]: no, this was an incorrect check originally which wasn't listsing the params associated with a profile.

Comment (srijeet0406) [inline traffic_ops/traffic_ops_golang/profile/profiles.go:444]: Could you pls use the `tc.PermParameterSecureRead` constant instead? That will avoid any future typos like this one.

**Review by srijeet0406 — APPROVED** [review]
LGTM, just waiting for the tp integration tests to pass.

**Pull request merged**